dist/autosize.js

```diff
@@ -144,14 +145,19 @@
 	        resize();
 	        actualHeight = computed.boxSizing === 'content-box' ? Math.round(parseFloat(window.getComputedStyle(ta, null).height)) : ta.offsetHeight;
 	      }
-	    } else {
-	      // Normally keep overflow set to hidden, to avoid flash of scrollbar as the textarea expands.
-	      if (computed.overflowY !== 'hidden') {
-	        changeOverflow('hidden');
-	        resize();
-	        actualHeight = computed.boxSizing === 'content-box' ? Math.round(parseFloat(window.getComputedStyle(ta, null).height)) : ta.offsetHeight;
+	    } // If the scrollHeight is higher than the styleHeight at this point, it means that the textearea is taller than its parent and a scrollbar appeared
+	    // Due to the scrollbar added to the parent, the textarea is now a little narrower and therefore the height needs to be updated accordingly
+	    else if (scrollHeight > styleHeight) {
+	        ta.style.height = scrollHeight + "px";
+	        actualHeight = computed.boxSizing === 'content-box' ? scrollHeight : ta.offsetHeight;
+	      } else {
```

Comment:
indent looks broken

Reply:
That's a generated file, I did not write that one. I updated the source code to use tabs instead of spaces but it did not change this particular file.

Reply:
ok